mmeval/metrics/precision_recall_f1score.py

```diff
@@ -112,7 +169,7 @@ def _precision_recall_f1_support(pred_positive: Union[np.ndarray,
     return precision, recall, f1_score, support
 
 
-class SingleLabelMetric(BaseMetric):
+class SingleLabelPrecisionRecallF1score(BaseMetric):
```

Comment:
Metric renaming is BC-breaking. You can refer to PR #83 to keep compatibility. Simply 2 steps:
1. Modify `__deprecated_metric_names__` in `mmeval.metrics.__init__.py`, so that we can `from mmeval import OldMetricClass`
2. add `OldMetricClass = NewMetricClass` at the bottom of each metric file, so that we can `from mmeval.metrics.old_metric_class import OldMetricClass`

Reply:
done